src/SymbolCollector.Console/Program.cs

```diff
@@ -183,7 +185,11 @@ static void Print(ObjectFileResult r)
                 var sorter = host.Services.GetRequiredService<Symsorter>();
 
                 await sorter.ProcessBundle(
-                    new SymsorterParameters(args.Path, args.BatchType, args.BundleId, args.DryRun),
+                    new SymsorterParameters(
+                        args.Path,
+                        args.BatchType?.ToString() ?? string.Empty,
```

Comment:
What about we blow up instead?
```suggestion
                        args.BatchType?.ToString() ?? throw new ArgumentException("BatchType required),",
```

Reply:
I wish the framework was a bit smarter just like it does with nullable string ...